src/actions/search.js

```diff
@@ -64,8 +64,7 @@ export function searchPosts(teamId, terms, isOrSearch = false, includeDeletedCha
                 type: SearchTypes.RECEIVED_SEARCH_TERM,
                 data: {
                     teamId,
-                    terms,
-                    isOrSearch,
+                    params,
```

Comment:
The reducer that handles this action still expects to the `terms` and `isOrSearch` fields, so you should either update the reducer to use `params` or just pass them with their previous names like
```javascript
data: {
    teamId,
    terms: params.terms,
    isOrSearch: params.is_or_search,
}
```